standards/collections.md

```diff
@@ -626,3 +626,54 @@ GET /articles?ordering=title,-reviewRating                  // ordering multiple
 GET /articles?orderings=title&orderings=-reviewRating       // never use pluralized "orderings"
 DELETE /articles?ordering=title                             // ordering only applies to GET methods.
 ```
+
+## ETags with Collections
+
+ETags in consideration with resource collections post some additional considerations on what the `ETag` should represent and in what granularity. For more general details about using `ETags` in request and response headers for optimistic concurrency, see the [Request Response](request-response.md#etag) section. When working with ETags in collections, the following considerations should be taken into account:
+
+- `ETags` used in a response header **MUST** represent the entire of the response body, which is the entire collection rather than an individual resource. This is typically less useful in many situations. Working with paged collection results means the response header `ETag` is page-sensitive.
```

Comment:
What about cases where we have a response that contains a collection of resources, but also "meta" data about the response such as pagination?  Would it be preferable to add an ETag as a sibling of the collection in the body instead of the response header?

Reply:
Ignore me, I only read it 3 times, lol.